src/collection.ts

```diff
@@ -613,6 +613,55 @@ export default class Collection {
     }) as Promise<KintoResponse<{ deleted: boolean }>>;
   }
 
+  /**
+   * Deletes records from the current collection.
+   *
+   * Sorting is done by passing a `sort` string option:
+   *
+   * - The field to order the results by, prefixed with `-` for descending.
+   * Default: `-last_modified`.
+   *
+   * @see http://kinto.readthedocs.io/en/stable/api/1.x/sorting.html
+   *
+   * Filtering is done by passing a `filters` option object:
+   *
+   * - `{fieldname: "value"}`
+   * - `{min_fieldname: 4000}`
+   * - `{in_fieldname: "1,2,3"}`
+   * - `{not_fieldname: 0}`
+   * - `{exclude_fieldname: "0,1"}`
+   *
+   * @see http://kinto.readthedocs.io/en/stable/api/1.x/filtering.html
+   *
+   * Paginating is done by passing a `limit` option, then calling the `next()`
```

Comment:
...`The default server pagination size can be forced by passing ... `


Also, isn't it automatic? I don't think we should mention the `calling the next() method` no?

Reply:
I'm not sure about this, but the `listRecords` function in the same file has a very similar documentation and is using the same `PaginatedParams` object as parameter. If I remove it I think we should also update the documentation of `listRecords`, for consistency sake.